style.css

```diff
@@ -1,24 +1,34 @@
+@import url("http://fonts.cdnfonts.com/css/cursed-timer-ulil");
```

Comment:
- [ ] Try and avoid defining styles for element selectors (`p`, `div`, `h1`, etc). Instead always use classnames.